superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -852,25 +852,36 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     setTabKey(key);
   };
 
-  const renderStepTwoAlert = () =>
-    db?.engine && (
-      <StyledAlertMargin>
-        <Alert
-          closable={false}
-          css={(theme: SupersetTheme) => antDAlertStyles(theme)}
-          type="info"
-          showIcon
-          message={
-            engineSpecificAlertMapping[db.engine]?.message ||
-            connectionAlert?.DEFAULT?.message
-          }
-          description={
-            engineSpecificAlertMapping[db.engine]?.description ||
-            connectionAlert?.DEFAULT?.description
-          }
-        />
-      </StyledAlertMargin>
+  const renderStepTwoAlert = () => {
+    const path = window.location.hostname;
```

Comment:
A comment here... a `path` has a well-defined meaning in the context of URLs, it's the part after `hostname:port`. Let's call this `hostname`, since it's what it is:

```suggestion
    const hostname = window.location.hostname;
```